res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+            </description>
+            <effects>
+                <effect trigger="happen" type="triggernews" time="2,5,6,18,22" news="cujo-news-roter-platz-07" />
+            </effects>
+            <data genre="4" price="0.85" quality="65" />
+        </news>
+        <news id="cujo-news-roter-platz-07" type="2" thread_id="cujo-news-roter-platz" creator="8827" created_by="Cujo">
+            <title>
+                <de>Schessnah-Pilot wird begnadigt</de>
+                <en>Schessnah pilot is pardoned</en>
+            </title>
+            <description>
+                <de>Der auf dem Roten Platz gelandete Schessnah-Pilot Markus Trust wird begnadigt. Nach 15 Monaten Arbeitslager kann er morgen zu seiner Familie zurückkehren. Seine Schessnah wird verschrottet.</de>
+                <en>Schessnah pilot Markus Trust, who landed in Red Square, is pardoned. After 15 months in a labor camp, he can return to his family tomorrow. His Schessnah will be scrapped.</en>
+            </description>
+			<effects />
+            <data genre="4" price="0.78" quality="55" />
+        </news>
+		<!-- "Ältester Mensch der Welt" -->
+        <news id="cujo-news-aeltestermensch-01" type="0" thread_id="cujo-news-aeltestermensch" creator="8827" created_by="Cujo">
+            <title>
+                <de>Ältester Mensch der Welt gestorben</de>
+                <en>Oldest person in the world died</en>
+            </title>
+            <description>
+                <de>Im Alter von 120 Jahren, 8 Monaten und 13 Tagen ist der Japaner Shigeko Izmari gestorben. Als er geboren wurde, produzierte der Konsolenhersteller Nidennto noch Gummistiefel.</de>
+                <en>At the age of 120 years, 8 months and 13 days, Japanese Shigeko Izmari has died. When he was born, the console manufacturer Nidennto was still producing rubber boots.</en>
+            </description>
+            <data genre="4" price="0.9" quality="30" />
+        </news>
+		<!-- "Bill Gates: Internet ist nur ein Hype" -->
+        <news id="cujo-news-internet-trend-01" type="0" thread_id="cujo-news-internet-trend" creator="8827" created_by="Cujo">
+            <title>
+                <de>Willy Bates: Internet ist nur ein Trend</de>
+                <en>Willy Bates: Internet is just a trend</en>
+            </title>
+            <description>
+                <de>In einem Fernseh-Interview prophezeit der Software-Entwickler Willy Bates dem Internet keine große Zukunft: "Das ist nur ein kurzfristiger Trend. In 10 Jahren gibt es das nicht mehr."</de>
+                <en>In a television interview, software developer Willy Bates doesn't predict a great future for the Internet: "It's just a short-term trend. It won't be around in 10 years."</en>
+            </description>
+            <data genre="3" price="0.8" quality="37" />
+            <availability year_range_from="1993" year_range_to="1995" />

```

Comment:
Eignet sich nicht für mehrfaches Senden

Reply:
Wie fast alle News :)

Reply:
Ich denke auch, nach 2 Jahren kann man die wieder rauskramen.
Gibt doch so manche Newssendung die alten Kram in ihrem "Ratgeberteil" aufwaermt ... 

Oder man denke an die Sommerlochnews...